include/picongpu/particles/atomicPhysics/ionizationPotentialDepression/StewartPyattIPD.hpp

```diff
@@ -211,7 +211,7 @@ namespace picongpu::particles::atomicPhysics::ionizationPotentialDepression
             // eV/(sim.unit.mass() * sim.unit.length()^2 / sim.unit.time()^2)
             constexpr float_X eV = static_cast<float_X>(
                 picongpu::sim.unit.mass() * pmacc::math::cPow(picongpu::sim.unit.length(), 2u)
-                / pmacc::math::cPow(picongpu::sim.unit.time(), 2u) * sim.si.conv.joule2ev(1.0));
+                / pmacc::math::cPow(picongpu::sim.unit.time(), 2u) * sim.si.conv().joule2ev(1.0));
```

Comment:
Misuse of a conversion function, what we actually want here the value of 1eV in internal PIC units, I would suggest to define something along the line
```c++
struct SIUnits
{
    ...
   //! get 1eV in J
   constexpr float_64 geteV()
   {
        return 1.602176634e-19;
    }
```
and replace the lines 212 -214 with the following.

```suggestion
     constexpr float_X eV = static_cast<float_X>(picongpu::sim.si.geteV() / piconggpu::sim.unit.energy());
```
or write a dedicated conversion method that converts internal energy to eV